tests/fields.js

```diff
@@ -258,3 +258,59 @@ test('badges', (t) => {
     'Should sort `badges[]`',
   )
 })
+
+test('pnpm', macro.sortObject, {
```

Comment:
If this test looks overly specific it's because I took a real world example from my own repository (nothing private here) and set the expectation of how we'd like it sorted.